### PR TITLE
That's 60s

### PR DESCRIPTION
**Pull request by BrianG61UK**

### CODE REVIEW

File: build-source/snp.index [hunk 1]
```diff
@@ -140,7 +141,6 @@ france4hd=france4
 france5hd=france5
 euronewsdeutsch=euronews
 realityxtra2=realityxtra2
-classichits=classichits
```
Comment (AbuBaniaz) [inline build-source/snp.index]: Please re-add this link.
Unless  service exists on your box it wont show. When it reverts back it will show automatically.

**Review by AbuBaniaz — CHANGES_REQUESTED** [review]
Please re-add the classic hits snp  link.
Unless  service exists on your box it wont show. When it reverts back it will show automatically.

**Comment by AbuBaniaz** [comment]
Bother, I missed the fact that you deleted the classichits logo too, please re-add it

**Comment by BrianG61UK** [comment]
"Classic Hits" logo is restored.

Yes, sorry. I assumed that since "Classic Hits" split into "That's 60s" and "That's 80s" it probably wasn't coming back.
But, you're right, that's only a guess.

**Pull request merged**